src/main/resources/tomcat-eureka-config/dbmigration/registry-changelog-eureka-config.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<!--
+  #%L
+  Eureka! Clinical Registry Service
+  %%
+  Copyright (C) 2017 - 2018 Emory University
+  %%
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+       http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  #L%
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                         http://www.liquibase.org/xml/ns/dbchangelog
+                         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd"
+		   logicalFilePath="changelog-master.xml">
+    <!--{% if eureka_webapp_installed is defined and eureka_webapp_installed %}-->
+    <changeSet author="arpost" id="EC-ANALYTICS-WEBAPP-1">
+	<insert tableName="COMPONENTS">
+	    <column name="ID" valueSequenceNext="COMPONENT_SEQ"/>
+	    <column name="NAME" value="Eureka! Clinical Analytics Webapp"/>
+	    <column name="DISPLAYNAME" value="Analytics Webapp"/>
+	    <column name="DESCRIPTION" value="API gateway for Eureka! Clinical Analytics"/>
+	    <column name="URL" value="{{eureka_webapp_server_base_url}}"/>
+	    <column name="TYPE_ID" valueComputed="(SELECT ID FROM COMPONENT_TYPES WHERE NAME='APIGATEWAY')"/>
+	</insert>
+    </changeSet>
+    <!--{% endif %}
+    {% if eureka_services_installed is defined and eureka_services_installed %}-->   
+    <changeSet author="arpost" id="EC-ANALYTICS-SERVICE-1">
+	<insert tableName="COMPONENTS">
+	    <column name="ID" valueSequenceNext="COMPONENT_SEQ"/>
+	    <column name="NAME" value="Eureka! Clinical Analytics Service"/>
+	    <column name="DISPLAYNAME" value="Analytics Service"/>
+	    <column name="DESCRIPTION" value="REST APIs for Eureka! Clinical Analytics"/>
+	    <column name="URL" value="{{eureka_services_base_url}}"/>
```

Comment:
All of the URLs for the registry entries need to be syntactically valid URLs, even if not all of these services will be running in embedded tomcat.

Reply:
These URL are generated from Ansible, do you want me to hardcode the URL here? If so, part of the url is {{public_hostname}}, which I cannot find in the provisioning project.

Or can you specify what need to be done?

Reply:
Yes, hardcode a localhost URL. If a component is installed in the embedded tomcat, use its actual URL. If not, use a URL like https://localhost:webclient_port/component_name.